components/raftstore/src/coprocessor/split_observer.rs

```diff
@@ -236,7 +235,7 @@ mod tests {
         assert!(resp.is_ok());
         assert!(!req.has_split(), "only split req should be handle.");
 
-        req = new_split_request(b"test");
```

Comment:
The previous test case is invalid because the raw key is not in the region.

Reply:
Does the following line have relations to the key?

Reply:
nope.